### DISCUSSION TITLE
How do you use getColors/setColors?

### DISCUSSION THREAD

**Discussion by mitya33** [epicmaxco/vuestic-ui#3543] [Q&A] [answered]
Hi all - I'm new to Vuestic and have been reading about customising colours. I can successfully override colours via config but I'm unsure how to use the above two methods. I had assumed they were exports from Vuestic, but doing this:

```
<script setup>
import { getColors } from 'vuestic-ui'
...
</script>
```

...results in an error ("amibuguous indirect export").

[This page](https://ui.vuestic.dev/services/colors-config#reactivity) discusses the `getColors()` method but doesn't actually show a full example of its usage. Does one exist?

Thank you.

**Comment by m0ksem** [comment, marked as answer]
Hi. You should use `useColors` composable from `vuestic-ui` in script setup.

```vue
<script setup>
import { useColors} from 'vuestic-ui'

const { colors, getColor, currentPresetName } = useColors()
</script>
```

**Reply by mitya33** [reply]
Thank you. I feel that could be better explained on the docs, however.

**Reply by m0ksem** [reply]
Would be great if you can help us improve the docs! If you feel that something must be changed in docs, please make an issue and describe how you want to improve the docs experience. Also notify us if you're planning to make a PR, or it is just an issue.
If you don't know how it can be better explained, just make us know what has poor explanation.
